### DISCUSSION TITLE
feat(experiments/evals): Regex based evaluators

### DISCUSSION THREAD

**Discussion by marliessophie** [langfuse/langfuse#4671] [Ideas]
### Describe the feature or potential improvement

Add support for langfuse-pre-build regex-based evaluators, for example: `validJson`, `exactMatch`, `listContains`

**Comment by sfarthin** [comment]
I suspect it might be helpful to provide "regex" UI experience for non-technical folk, but underneath the hood, I suspect execution would be the same as the "code" evals: https://github.com/orgs/langfuse/discussions/6087

**Comment by clemra** [comment]
This is now live: https://langfuse.com/changelog/2026-05-28-code-evaluators
